packages/dds/tree/src/test/utils.ts

```diff
@@ -1172,7 +1172,6 @@ export function treeTestFactory(
 			}),
 		options.attributes ?? new SharedTreeFactory().attributes,
 		options.options ?? { jsonValidator: typeboxValidator },
-		options.telemetryContextPrefix ?? "SharedTree",
```

Comment:
Seems like we should keep passing `options.telemetryContextPrefix` here (no coalescing) so if anything is passed in the property bag it still applies? Otherwise that bit of the property bag is not necessary anymore.